docs/blog/2025-community-update.rst

```diff
@@ -0,0 +1,91 @@
+:og:image: https://www.writethedocs.org/_static/logo-opengraph.png
+
+.. post:: Aug 8, 2025
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - August 2025
+=============================================
+
+Hey there documentarians!
+
+Eric Holscher here, sharing this year's community update as we reflect on another year of growth and change for Write the Docs.
+
+We've been heavily focused on conferences this year, but our Slack community, Newsletter, Meetups, and other initiatives continue without too much change. 
```

Comment:
```suggestion
We've been heavily focused on conferences this year, but our Slack community, Newsletter, Meetups, salary survey, and other initiatives continue without major changes. 
```

Reply:
This isn't a full team, so feels odd to mention here? I think it's covered in other initiatives?